caladrius/sint_maarten_2017.py

```diff
@@ -303,6 +301,39 @@ def splitDatapoints(filepath):
     return split_mappings
 
 
+def createInferenceDataset():
```

Comment:
I still don't exactly understand what inference is doing differently than test? Except that it is done on images that are both in before and after folder? 
If there is no other difference, it might make more sense to make a separate folder for it (could do in sint_maarten.py as being done now) but not make a special model-type for it?

Reply:
Inference does not contain labels. It is the dataset split for the image pairs which we want to predict on but do not have the true values for.